docker/ubuntu-2204-builder.Dockerfile

```diff
@@ -72,7 +72,8 @@ RUN <<EOT bash
     rm \$NODEPATH.tar.xz
     mv \$NODEPATH node-v20.11.0
     export PATH="/opt/node-v20.11.0/bin:$PATH"
-    npm i -g yarn
+    corepack enable
+    yarn set version stable
```

Comment:
Does this try to write anything? [The docs](https://yarnpkg.com/cli/set/version#details) seem to indicate it will try to update some package.json at the root of the project, so what will it do in this case given that it's being run from `/opt`?